jbpm-runtime-manager/src/main/java/org/jbpm/runtime/manager/impl/lock/AbstractRuntimeManagerLockStrategy.java

```diff
@@ -0,0 +1,136 @@
+/*
+ * Copyright 2020 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.jbpm.runtime.manager.impl.lock;
+
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Set;
+
+import org.jbpm.runtime.manager.spi.RuntimeManagerLock;
+import org.jbpm.runtime.manager.spi.RuntimeManagerLockFactory;
+import org.jbpm.runtime.manager.spi.RuntimeManagerLockStrategy;
+import org.kie.api.runtime.manager.RuntimeEngine;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+
+public abstract class AbstractRuntimeManagerLockStrategy implements RuntimeManagerLockStrategy {
+
+    protected static final Logger logger = LoggerFactory.getLogger(AbstractRuntimeManagerLockStrategy.class);
+
+    protected Map<Long, RuntimeManagerLockThreadsInfo> engineLocks; 
+
+    private RuntimeManagerLockFactory runtimeManagerLockFactory;
+
+    
+    public AbstractRuntimeManagerLockStrategy() {
```

Comment:
As it is abstract, it shouldn't be public
```suggestion
    protected AbstractRuntimeManagerLockStrategy() {
```

Reply:
done